x-pack/plugins/apm/server/routes/traces/get_trace_items.ts

```diff
@@ -54,6 +55,7 @@ export async function getTraceItems(
       events: [ProcessorEvent.span, ProcessorEvent.transaction],
     },
     body: {
+      track_total_hits: maxTraceItems + 1,
```

Comment:
Why is `+1` necessary here?

Reply:
Below we do:
```ts
  const exceedsMax = traceResponse.hits.total.value > maxTraceItems;
```

I want to make sure that the `total.value` is indeed larger than `maxTraceItems` when there are more items than `maxTraceItems`.